frontend/src/app/offers/offer-detail/offer-detail.component.ts

```diff
@@ -24,7 +24,12 @@ export class OfferDetailComponent implements OnInit {
 
   ngOnInit() {
     this.offer$ = this.activatedRoute.params
-    .switchMap(params => this.offersService.getOffer(params.offerId));
+    .switchMap(params => this.offersService.getOffer(params.offerId))
+    .map(offer$ => {
+      if (offer$.image === null) {
+        offer$.image = `../../assets/img/offer/offer_${Math.round(Math.random() * 3) + 1}.jpg`;
```

Comment:
we decided to stick with one image (volontulo baner)

Reply:
don't use `../../`

Reply:
There is a baseUrl defined in compilerOptions in tsconfig.json file, so when I use absolute path it still will be working corectly. but I'm quite sure you wanted more sophisticated change here.